pages/linux/imgp.md

```diff
@@ -0,0 +1,15 @@
+# imgp
+
+> Command line image resizer and rotator for JPEG and PNG images.
+
+- Convert some images and directories:
```

Comment:
We should clarify what does "convert" do ? Does it just resize to the target resolution ? And also what does "images and directories" mean ? In this context, it means we are converting directories too, which does not make sense.